kubernetes/resource_kubernetes_persistent_volume_test.go

```diff
@@ -833,6 +858,54 @@ resource "aws_ebs_volume" "test" {
 `, name, zone, diskName)
 }
 
+func testAccKubernetesPersistentVolumeConfig_azure_basic(name, diskName, region string) string {
+	return fmt.Sprintf(`
+	resource "kubernetes_persistent_volume" "test" {
+		metadata {
+			annotations {
+				TestAnnotationOne = "one"
+				TestAnnotationTwo = "two"
+			}
+	
+			labels {
+				TestLabelOne   = "one"
+				TestLabelTwo   = "two"
+				TestLabelThree = "three"
+			}
+	
+			name = "%s"
+		}
+		spec {
+			capacity {
+				storage = "2Gi"
+			}
+			access_modes = ["ReadWriteOnce"]
+			persistent_volume_source {
+				azure_disk {
+					caching_mode = "None"
+					data_disk_uri = "${azurerm_managed_disk.dbstorage.id}"
+					disk_name = "%s"
+					kind = "Managed"
+				}
+			}
+		}
+	}
+	resource "azurerm_managed_disk" "dbstorage" {
+		name                 = "%s"
+		location             = "West US 2"
```

Comment:
This will cause the test to fail if run in any other location. 
You should read the location from a variable called `location` which our CI populates appropriately when running the test suite.